src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/WindowsForms/MyAppFileAccessor.cs

```diff
@@ -13,8 +13,10 @@ internal class MyAppFileAccessor : IMyAppFileAccessor, IDisposable
     private readonly IServiceProvider _serviceProvider;
     private readonly UnconfiguredProject _project;
     private readonly IProjectThreadingService _threadingService;
+    private readonly IPhysicalProjectTreeStorage _storage;
     private DocData? _docData;
     private MyAppDocument? myAppDocument;
+    private string _absolutePath;
```

Comment:
This field can be readonly.

That said, I don't think we need to retain a reference to this path unless `TryGetMyAppFileAsync` is called very often for a given project.